persistence/src/commonTest/kotlin/com/wire/kalium/persistence/dao/message/MessageDAOTest.kt

```diff
@@ -928,21 +929,23 @@ class MessageDAOTest : BaseDatabaseTest() {
         // then
         val updatedMessage = messageDAO.getMessageById(messageId, conversationId).firstOrNull()
         val updatedMessageContent = updatedMessage?.content
+
+        // assert values that should not be updated
```

Comment:
```suggestion
        // asset values that should not be updated
```

Reply:
Maybe move this comment to the Line 936 as `visibility` field should still be updated as it is not an asset-specific field.

Reply:
I don't think visibility will be updated